core/controllers/admin.py

```diff
@@ -346,10 +325,12 @@ def get(self) -> None:
             )
         ]
 
-        config_properties = config_domain.Registry.get_config_property_schemas()
-
+        # Remove the config_properties from here once the classroom
```

Comment:
@Nik-09 Could you format this as `# TODO(...): ...`?

Also maybe start it with `Remove the ``config_properties`` key-value pair from this dict, once ...`

Reply:
Done

Reply:
Thanks @Nik-09. Could you please  clarify the description of #19810 (and this) as well? I don't know what you mean by "once classroom config and new platform parameter tabs are ready to onboard the existing config fields." -- that doesn't really make sense to me. You might want to try expressing it a different way.